physionet-django/project/models.py

```diff
@@ -4,40 +4,36 @@
 import os
 import shutil
 import uuid
-import pdb
 import pytz
-import stat
 import logging
 from distutils.version import StrictVersion
 
-import bleach
-import ckeditor.fields
 from html2text import html2text
 from django.conf import settings
 from django.contrib.contenttypes.fields import GenericForeignKey, GenericRelation
 from django.contrib.contenttypes.models import ContentType
-from django.contrib.staticfiles.templatetags.staticfiles import static
 from django.core.validators import MaxValueValidator, MinValueValidator
-from django.contrib.auth.hashers import check_password, make_password
-from django.db import models, DatabaseError, transaction
+from django.db import models, transaction
 from django.forms.utils import ErrorList
 from django.urls import reverse
 from django.utils import timezone
 from django.utils.html import format_html, strip_tags
 from django.utils.text import slugify
 from background_task import background
-from django.utils.crypto import get_random_string
 
+from project.modelcomponents.authors import (AuthorInvitation, Affiliation, PublishedAffiliation,
```

Comment:
Not a big deal but slightly bothersome that any new classes in the modelcomponents modules will need to be explicitly imported here (this may not be immediately obvious to people when updating the modules) but I guess (1) there's not much we can do about that other than maybe making a note somewhere and (2) those modules should be relatively stable once the refactoring is done anyway.

Reply:
We could also `import *` for these particular instances. Might be more maintainable. What do you think?